File: Java/Vetores/soma_vetores.java
// Exercício: Soma de vetores
// Objetivo: somar os elementos de dois vetores, posição por posição, armazenando o resultado em um terceiro vetor.

import java.util.Scanner;

public class soma_vetores {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);

        int n;

        System.out.print("Quantos valores vai ter cada vetor? ");
        n = sc.nextInt();

        System.out.println("Digite os valores do vetor A: ");
        int[] vetA = new int[n];
        for (int i = 0; i < n; i++) {
            vetA[i] = sc.nextInt();
        }

        System.out.println("Digite os valores do vetor B:");
        int[] vetB = new int[n];
        for (int i = 0; i < n; i++) {
            vetB[i] = sc.nextInt();
        }

        int[] vetC = new int[n];
        for (int i = 0; i < n; i++) {
            vetC[i] = vetA[i] + vetB[i];
        }

        System.out.println("VETOR RESULTANTE: ");
        for (int i = 0; i < n; i++) {
            System.out.println(vetC[i]);
        }

        sc.close();
    }
}

File: Python/vetores/soma_vetores.py
# Exercício: Soma de vetores
# Objetivo: armazenar dois vetores, somar seus elementos correspondentes e armazenar o resultado em um terceiro vetor.

n = int(input("Quantos valores vai ter cada vetor? "))

vetA = [0 for x in range(n)]
vetB = [0 for x in range(n)]
vetC = [0 for x in range(n)]

print("Digite os valores do vetor A: ")
for i in range(n):
    vetA[i] = int(input())

print("Digite os valores do vetor B: ")
for i in range(n):
    vetB[i] = int(input())

for i in range(n):
    vetC[i] = vetA[i] + vetB[i]

print("VETOR RESULTANTE:")
for i in range(n):
    print(vetC[i])             